Remora.Discord.Commands/Extensions/CommandTreeExtensions.cs

```diff
@@ -431,6 +438,37 @@ CommandNode command
             return Result<IReadOnlyList<IApplicationCommandOption>>.FromSuccess(parameterOptions);
         }
 
+        private static Result<Optional<IReadOnlyList<ChannelType>>> CreateChannelTypesOption(
```

Comment:
Your formatting is a little off here - look at the related methods for an example of how to properly do it.